warp/payload/payload_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package payload
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAddressedPayload(t *testing.T) {
+	require := require.New(t)
+
+	addressedPayload, err := NewAddressedPayload(
+		ids.GenerateTestID(),
+		ids.GenerateTestID(),
+		[]byte("payload"),
+	)
+	require.NoError(err)
+
+	addressedPayloadBytes := addressedPayload.Bytes()
+	addressedPayload2, err := ParseAddressedPayload(addressedPayloadBytes)
+	require.NoError(err)
+	require.Equal(addressedPayload, addressedPayload2)
+}
+
+func TestParseAddressedPayloadJunk(t *testing.T) {
+	_, err := ParseAddressedPayload(utils.RandomBytes(1024))
+	require.Error(t, err)
```

Comment:
can we keep the tests consistent with `TestAddressedPayload` by adding this line `require := require.New(t)`

Reply:
I'd prefer a two line to a three line test here since it's not going to be re-used at all.